### PR TITLE
Update CRI testing policy

### PR DESCRIPTION
**Pull request by yujuhong**
Change the layout in the test grid.

### CODE REVIEW

File: contributors/devel/cri-testing-policy.md [hunk 1]
```diff
@@ -50,16 +50,16 @@ To publish tests results, please submit a proposal in the
...
-These test results should be published under the `sig-node` tab, grouped by the
-runtimes, organized as follows.
+These test results should be published under the `sig-node` tab, organized
+as follows.
 
 ```
-sig-node -> sig-node-{Kubernetes-version} -> sig-node-{runtime-name} -> [page containing all test jobs]
-```
+sig-node -> sig-node-cri-{Kubernetes-version} -> [page containing the required jobs]
+``
```
Comment (spiffxp) [inline contributors/devel/cri-testing-policy.md]: nit: missing a `, throws off formatting below

File: contributors/devel/cri-testing-policy.md [hunk 2]
```diff
@@ -50,16 +50,16 @@ To publish tests results, please submit a proposal in the
...
 
-These test results should be published under the `sig-node` tab, grouped by the
-runtimes, organized as follows.
+These test results should be published under the `sig-node` tab, organized
+as follows.
 
 ```
-sig-node -> sig-node-{Kubernetes-version} -> sig-node-{runtime-name} -> [page containing all test jobs]
-```
+sig-node -> sig-node-cri-{Kubernetes-version} -> [page containing the required jobs]
```
Comment (spiffxp) [inline contributors/devel/cri-testing-policy.md]: it's unclear to me what 'required' means here, maybe a sample list of required jobs?

Comment (yujuhong) [reply contributors/devel/cri-testing-policy.md]: This is described in the previous section: node conformance & node feature suite.

**Comment by dchen1107** [comment]
/approve

cc/ @timothysc since he recently raised the questions on CRI validation tests.

**Comment by spiffxp** [comment]
/approve
per the above /approve

**Pull request merged**